.circleci/config.yml

```diff
@@ -123,6 +123,15 @@ jobs:
       - run: echo 're-enable tests here'
       - run: echo 'make utils.test'
 
+  check-vulnerabilities:
+    <<: *python_node_postgres_images
+    steps:
+      - *attach_workspace
+      - run: make admin.setup-dev
+      - run: make api.setup-dev
+      - run: make utils.setup-dev
```

Comment:
Is this not just `- run: make setup-dev`?

Reply:
`make setup-dev` will also run for the `status` directory which isn't needed for now

Reply:
OK no worries.